README.md

```diff
@@ -29,16 +28,10 @@ If you just want to see see it work:
 * `frame`, an array of 50 RGB values to send to the Holiday
 * `callback`, an optional callback function passed to `Socket.send`
 
-If `callback` isn't supplied and an error occurs (e.g. 
-`getaddrinfo ENOTFOUND`), the `Holiday` will emit `error`. 
+If `callback` isn't supplied and an error occurs (e.g. `getaddrinfo ENOTFOUND`), the `Holiday` will emit `error`.
 
-**WARNING:** On OS X, if a `setInterval` timer calls `send` an `error` 
-emitted to no subscribers *should* bring down Node if you've called `unref`,
-but it doesn't. 
+**WARNING:** On macOS, if a `setInterval` timer calls `send` an `error` emitted to no subscribers *should* bring down NodeJS if you've called `unref`, but it doesn't.
 
 ## Testing
 
-Once you've found your Holiday, try `bin/thrash.js`. If it works and you're
-into red, white, and green themed holidays at the end of the year, try
-`bin/xmas.js`. The latter has a simple animation module hiding in it, waiting
-to be broken out.
+Once you've found your Holiday, try `node bin/thrash.js <HOLIDAY IP> &`. If it works and you're into red, white, and green themed holidays at the end of the year, try `node bin/xmas.js <HOLIDAY IP> &`. The latter has a simple animation module hiding in it, waiting to be broken out.
```

Comment:
Did I forget to make those executable?

I'm happy with either `bin/x.js` or `node bin/x.js` but I'd prefer not to advise putting it in the background in case the reader doesn't know what to do next.

Reply:
No, you are correct - [they are set](https://github.com/garthk/holiday-udp/blob/12bd64b04d33bb5c4bce513b92e01a2b9d5c2b60/bin/thrash.js#L1) to be executable.

I've not create a [NPM/NodeJS] dependency before, but I was thinking by [reverting](https://github.com/garthk/holiday-udp/commit/c4cadf2a91bb5cd55fca7559e8d7aa6b06958c79) "optimist" back to a `devDependencies` then when this project is called/installed from other projects, "optimist" wouldn't be installed into those projects - which would not be adding this project for the trash and xmas (🇮🇹 Italian Flag 😜) scripts, they'd nearly always be adding this project for the UDP API `send()` function.

By making `node ` the script calling prefix of the command [to me] it makes it seem more like a "dev" command - hinting that the "devDependencies" install is needed.